consumer-features/example-hello-world/v3/example-hello-world-js-jest-pact/tests/consumer/consumer.test.js

```diff
@@ -0,0 +1,47 @@
+// Pact annotated code block - Setting up the Consumer
+const { pactWith } = require("jest-pact/v3");
+const { BearApiClient } = require("../../src/consumer");
+
+// Pact annotated code block - Setting up the Consumer
```

Comment:
Same "Setting up the Consumer" tag twice, I think trying to start twice without the end might do some funky things...but also thinking about it, nothing stopping from having that to also capture the close of the brackets at the end.
How about if I change the scraping so it puts them into a list rather than just one
e.g. now it constructs something like
```
code_blocks = {
 "Setting up the Consumer": "<whatever is in the block>"
 "Defining the pact, and calling the consumer": "<whatever is in the block>"
}
```

That could go to e.g.
`// Pact annotated code block - Setting up the Consumer - 1`
So then it's constructing:

```
code_blocks = {
 "Setting up the Consumer": [
  "<whatever is in the block>",
  "<then this block>",
 ]
 "Defining the pact, and calling the consumer": [
  "<whatever is in the block, only one here>"
 ]
}
```